boards/risc-v/k210/maix-bit/configs/kostest/defconfig

```diff
@@ -45,4 +45,5 @@ CONFIG_START_MONTH=2
 CONFIG_START_YEAR=2020
 CONFIG_TASK_NAME_SIZE=20
 CONFIG_TESTING_OSTEST=y
+CONFIG_TESTING_OSTEST_FPUTESTDISABLE=y
```

Comment:
why disable fpu test?

Reply:
Need addtion board level  arch_getfpu implementation etc.

Reply:
It doesn't need anymore after https://github.com/apache/incubator-nuttx/pull/6080.

Reply:
Got it